snappy/services.go

```diff
@@ -322,15 +277,15 @@ WantedBy={{.ServiceSystemdTarget}}
 	return templateOut.String()
 }
 
-func GenSocketFile(desc *ServiceDescription) string {
+func GenSocketFile(appInfo *snap.AppInfo) string {
 	serviceTemplate := `[Unit]
-Description={{.Description}} Socket Unit File
+Description=socket {{.App.Name}} for snap {{.App.Snap.Name}} - autogenerated DO NO EDIT
```

Comment:
`Socket {{.App.Name}} for snap {{.App.Snap.Name}}`

Capital-cased like other service/socket file, and no need to say auto-generated. These files usually come inside the package, and people are used to not fiddling with them.